foca/foca.py

```diff
@@ -0,0 +1,57 @@
+"""FOCA class definition"""

```

Comment:
This needs to be adapted according to the comments below. Also please add a `.` at the end as is generally done in the Python docstring conventions document [PEP257](https://www.python.org/dev/peps/pep-0257/).